### DISCUSSION TITLE
@mantine/core: MantineProvider was not found in component tree, make sure you have it in your app

### DISCUSSION THREAD

**Discussion by monira-ahmed** [mantinedev/mantine#5469] [Q&A]
"@mantine/core": "^7.3.2",
 "@mantine/dates": "^7.3.2",
 "@mantine/form": "^7.3.2",
 "@mantine/hooks": "^7.3.2",

**Comment by Toshinaki** [comment]
https://react.dev/reference/react/useContext#passing-data-deeply-into-the-tree
https://react.dev/reference/react/useContext#my-component-doesnt-see-the-value-from-my-provider
